docs/ovs_communication.adoc

```diff
@@ -65,9 +85,31 @@ TBD - XiaoDong
 
 . when and who will initialize OVS? What are the initialization steps
 .. br-int, br-tun are created during init time, and recreated later in RPC loop if needed
-.. need to determine the minimal initialization steps
-. what happen when there are multiple physical NICs on the system, which NIC do we pick to hook up to br-tun, br-vlan, br-ex (if needed)?
+A: for the 1st questions:
```

Comment:
we need to close on one of the option with justification :)

Reply:
for 1st questions, a-d answers are one option, they have to be done in once all.

Reply:
to answer your comments:
for 1st question, a-d are in one call
for 2nd question, vxlan_sys_4789 etc is for br-vlan decap/encap